Automattic/jetpack
# Add testimonial to backup connect screen

Add testimonial to backup connect screen

## Proposed changes:

* Adds new testimonials component to js-packages/components
* Uses new component on backup connect screen

### Other information:


## Jetpack product discussion

P2: p1HpG7-kd7-p2
Figma: bSsQjyTPK8VIfopwI4fdIU-fi-3005_2665

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch locally or with the Jetpack Beta plugin
2. DONT connect Jetpack to your wordpress account
3. Install the Jetpack VaultPress Backup plugin
4. If you are not redirected there, go to `/wp-admin/admin.php?page=jetpack-backup`
5. You should see the new testimonial underneath the first module
![image](https://github.com/Automattic/jetpack/assets/65001528/f46d8c66-55ee-4641-972e-faea09cba0bd)

6. Make sure it looks good on Mobile as well
![image](https://github.com/Automattic/jetpack/assets/65001528/4449c526-458e-4540-86b3-597a21a69e1e)
![image](https://github.com/Automattic/jetpack/assets/65001528/b2067492-20a2-49f8-95c3-21aa8a42e048)

7. Try out the left and right arrows on the module to make sure it goes through the quotes (right now there are only 2) 
![image](https://github.com/Automattic/jetpack/assets/65001528/3d2631c4-72bd-473a-b0ac-416daa2ea8dd)
